### PR TITLE
IGNITE-11314

### PR DESCRIPTION
**Pull request by sanpwc**
PR for TC tests.

### CODE REVIEW

File: modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+        stmt = conn.createStatement();
+
+        assert stmt != null;
+        assert !stmt.isClosed();
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void afterTest() throws Exception {
+        if (stmt != null && !stmt.isClosed()) {
```
Comment (AMashenkov) [inline modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java]: Simple U.closeQuiet(..) can be used here.

File: modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+        return res;
+    }
+
+    /**
+     * Check that queries goes to the same node within transaction.
+     *
+     * @throws Exception If failed.
+     */
+    @org.junit.Test
```
Comment (AMashenkov) [inline modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java]: Let's add import for Test annotation.

File: modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java (hunk 1 above)
Comment (sanpwc) [reply modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java]: fixed

File: modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java (hunk 2 above)
Comment (sanpwc) [reply modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBestEffortAffinityTransactionsSelfTest.java]: fixed

**Comment by dspavlov** [comment]
IGNITE-11314 | Resolved
-- | --

**Pull request closed without merging**